.azure-pipelines/packages.yml

```diff
@@ -7,12 +7,60 @@ pr: none
 variables:
   buildConfiguration: debug
   packageFeed: /ffc32c57-3e0e-4e8f-8633-a7ad01df2e45
+  dotnetCoreSdkVersion: 2.2.300
 
 jobs:
 
-#### Windows
+#### Nuget packages
 
-- job: windows
+- job: nuget
```

Comment:
One thing I'm thinking of is perhaps only splitting off the NuGet publishing steps and MSI publishing steps into separate jobs, since a lot of these steps are duplicated. Do you know if that would be possible? And what do you think of that approach?

Reply:
Good point. How about this: 9bcae6af0db58786aa896987e20ed7386507a6eb? I merged nuget and msi back into a single job, but only run the 2 nugets steps in one of the two "branches."

Reply:
Yup that looks good!